orbit/cmd/desktop/desktop.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/oklog/run"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
+	paniclog "github.com/virtuald/go-paniclog"
```

Comment:
Are we sure we want to introduce this new dependency? It may do what we need but it hasn't seem an update in 5 years and has 9 stars. Since it's basically just a single function (with some per-platform implementation details), I wonder if we shouldn't just copy it in our codebase (keeping it's original license, it's MIT). We could even remove some of the code as we don't care about undoing the redirect in our case. This would allow us to move faster should we find issues with it on our various platforms.